# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../ncurses/install-no-rm.patch
# Copyright (C) 2004 - 2011 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

One must not rm a library - especially not when the system - e.g. bash -
depends on it ...

  - Rene Rebe <rene@exactcode.de>

--- ncurses-5.9/mk-1st.awk.vanilla	2011-07-15 11:25:37.000000000 +0200
+++ ncurses-5.9/mk-1st.awk	2011-07-15 11:27:01.000000000 +0200
@@ -118,9 +118,9 @@
 function symlink(src,dst) {
 		if ( src != dst ) {
 			if ( SymLink !~ /.*-f.*/ ) {
-				printf "rm -f %s; ", dst
+				#printf "rm -f %s; ", dst
 			}
-			printf "$(LN_S) %s %s; ", src, dst
+			printf "$(LN_S) -f %s %s; ", src, dst
 		}
 	}
 function rmlink(directory, dst) {
